packages/babel-helper-define-polyfill-provider/src/normalize-options.ts

```diff
@@ -5,7 +5,7 @@ import type {
   MissingDependenciesOption,
 } from "./types";
 
-function patternToRegExp(pattern: Pattern): RegExp | undefined | null {
+function patternToRegExp(pattern: Pattern): RegExp | null {
```

Comment:
Thanks for handling the Flow Maybe type refinement!

Just wanted to confirm for this line and a few others -- was removing `undefined` intentional as opposed to removing `null`? Just thought I'd double check in code review since you had said [the reverse](https://github.com/babel/babel-polyfills/pull/121#discussion_r835882979) before

Reply:
Yeah, it looks like in many places the code explicitly passes `null` to mean "no value", rather than just not passing/returning anything.